packages/cart-sdk/hooks/use-checkout.ts

```diff
@@ -151,12 +151,10 @@ function useStandardCheckout() {
       const localCheckout = getLocalCheckout();
       let checkoutUrl: string | null = null;
       if (localCheckout != null) {
-         try {
-            const checkout = await updateCheckout(localCheckout.id, lineItems);
-            if (checkout != null) {
-               checkoutUrl = checkout.webUrl;
-            }
-         } catch (error) {}
```

Comment:
Any idea why this catch was here to begin with?

I don't think we should drop it without knowing why it was added and if that's still sensible.

Reply:
This was the pull that added the `catch` https://github.com/iFixit/react-commerce/pull/1011

Reply:
Thanks for finding the commit. I think you're right, we should keep the catch-all so that we don't skip creating a new checkout if the update fails.

Since we're doing our best to handle API errors in the storefront client, we should still hear about interesting updateCheckout failures, and the catch-all will filter out network failures and weird browser incompatibilities.